extension-api/package.json

```diff
@@ -10,11 +10,12 @@
   },
   "author": "David Woodruff",
   "license": "Apache-2.0",
-  "dependencies": {
+  "peerDependencies": {
     "aws-sdk": "^2.188.0"
```

Comment:
By moving aws-sdk to a peer dependency, we can avoid having a million versions of aws-sdk running around inside of nested node_modules, which'll speed up our installs and startups. By like, a lot, because aws-sdk is HEAVY.